File: DicodingSubmission1BFAA/app/src/main/java/com/example/tes/dicodingsubmission1bfaa/SplashActivity.java
package com.example.tes.dicodingsubmission1bfaa;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;

import com.example.tes.dicodingsubmission1bfaa.activity.UserListActivity;
import com.wang.avi.AVLoadingIndicatorView;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        ImageView logo = findViewById(R.id.logo);
        AVLoadingIndicatorView avi = findViewById(R.id.avi);
        avi.setIndicator("BallClipRotateMultipleIndicator");
        new Handler().postDelayed(() -> {
            Intent intent= new Intent(SplashActivity.this, UserListActivity.class);
            startActivity(intent);
            finish();
        },3000);
        Animation myanim = AnimationUtils.loadAnimation(this,R.anim.splashanimation);
        logo.startAnimation(myanim);
    }
}

File: GithubUserSubmission3/app/src/main/java/com/example/githubusersubmission3/presentation/SplashActivity.kt
package com.example.githubusersubmission3.presentation

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import com.example.githubusersubmission3.R
import com.example.githubusersubmission3.Utility

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        Handler().postDelayed({
            startActivity(Intent(this, MainActivity::class.java))
            Utility.transitionAct(this)
            finish()
        }, 3000)
    }
}